ppconsul.sh

```diff
@@ -1,32 +1,25 @@
 package: Ppconsul
-version: 0.0.1
-tag: 8ade80d0528b563d4b58bc4f09815fc1e3d5be19
+version: 0.0.2
+tag: f39961cdcfddc630616658fc9c1e833b51cba21b
 source: https://github.com/oliora/ppconsul
 requires:
   - boost
   - "GCC-Toolchain:(?!osx)"
 build_requires:
   - CMake
 ---
-#!/bin/sh
+#!/bin/bash -e
 
-cmake $SOURCEDIR                                              \
-      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                     \
-      -DBUILD_SHARED_LIBS=ON                                  \
-      ${BOOST_VERSION:+-DBOOST_ROOT=$BOOST_ROOT}              \
+cmake $SOURCEDIR                                 \
+      ${CMAKE_GENERATOR:+-G "$CMAKE_GENERATOR"}  \
```

Comment:
what is this ?

Reply:
By default cmake uses Makefiles as generator (meaning it generates stuff that can be compiled through `make`). With this option we open the possibility to use, for instance, `ninja`.